tests/file_lists/test_release_files.sh

```diff
@@ -17,15 +17,15 @@
 # This script should be used inside the release image to check expected file contents
 
 if [ "$#" -ne 1 ]; then
-    printf --  "ERROR: Missing script argument PYTHON_DISABLE. Please pass 0 or 1 to the script.\n"
+    printf --  "ERROR: Missing script argument debug_bazel_flags. Please pass it to the script.\n"
```

Comment:
Not sure if this help is actually helpful. What is `debug_bazel_flags`? What are the options and in what format I should pass it?